pkg/controllers/job/plugins/distributed-framework/pytorch/pytorch.go

```diff
@@ -0,0 +1,188 @@
+package pytorch
+
+import (
+	"flag"
+	"fmt"
+	"strconv"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/klog"
+
+	batch "volcano.sh/apis/pkg/apis/batch/v1alpha1"
+	"volcano.sh/volcano/pkg/controllers/job/helpers"
+	pluginsinterface "volcano.sh/volcano/pkg/controllers/job/plugins/interface"
+)
+
+const (
+	// PytorchPluginName is the name of the plugin
+	PytorchPluginName = "pytorch"
+	// DefaultPort is the default port for pytorch
+	DefaultPort = 23456
+	// DefaultMaster is the default task name of master host
+	DefaultMaster = "master"
+	// DefaultWorker is the default task name of worker host
+	DefaultWorker = "worker"
+
+	// EnvMasterPort is the env name of master port
+	EnvMasterPort = "MASTER_PORT"
+	// EnvMasterAddr is the env name of master addr
+	EnvMasterAddr = "MASTER_ADDR"
+	// EnvWorldSize is the env name of world size
+	EnvWorldSize = "WORLD_SIZE"
+	// EnvWorldSize is the env name of rank
+	EnvRank = "RANK"
+)
+
+type pytorchPlugin struct {
```

Comment:
It is recommended to change to Plugin. Otherwise, ci will report an error.
https://github.com/volcano-sh/volcano/pull/2354

Reply:
ok, l will change it later

Reply:
Sorry, This is the error I caused. If this struct is not public, will not encounter this error

Reply:
You don't have to change